unit/src/MethaneFluidPropertiesTest.C

```diff
@@ -149,4 +149,13 @@ MethaneFluidPropertiesTest::derivatives()
   ABS_TEST("mu", mu, _fp->mu(rho, T), 1.0e-15);
   ABS_TEST("dmu_dp", dmu_drho, dmu_drho_fd, 1.0e-15);
   REL_TEST("dmu_dT", dmu_dT, dmu_dT_fd, 1.0e-6);
+
+  // Henry's constant
+  T = 300.0;
+
+  Real dKh_dT_fd = (_fp->henryConstant(T + dT) - _fp->henryConstant(T - dT)) / (2.0 * dT);
+  Real Kh = 0.0, dKh_dT = 0.0;
+  _fp->henryConstant_dT(T, Kh, dKh_dT);
+  ABS_TEST("henry", Kh, _fp->henryConstant(T), 1.0e-12);
```

Comment:
Looks like this is too tight for our linux test box. You might have a similar issue in `CO2FluidPropertiesTest.C`.